discovery-provider/alembic/versions/5add54e23282_add_stems_table.py

```diff
@@ -0,0 +1,28 @@
+"""add stems support
+
+Revision ID: 5add54e23282
+Revises: b3084b7bc025
+Create Date: 2020-05-01 16:36:34.237413
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5add54e23282'
+down_revision = 'b3084b7bc025'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('stems',
+        sa.Column('parent_track_id', sa.Integer(), nullable=False, index=True),
```

Comment:
@piazz @raymondjacobson @jowlee still think we should waitto index these

Reply:
why?

Reply:
@raymondjacobson https://github.com/AudiusProject/audius-protocol/pull/419/files#r421120994

Reply:
i don't think it would give us any benefits at this point. it's very likely an index would use memory resources with no additional value

Reply:
we have to support queries from both sides and since we have to check this table a lot, I think it makes sense to run with indexes? It's not going to be huge in memory / do we see that we're suffering from memory pressure?

Reply:
especially in joins. my vote is to keep them in. if we see a ton of memory pressure from just this change, i think we should reconsider other stuff.

Reply:
this is a slippery slope where we add indexes arbitrarily cause we think it will increase performance when in reality it may not do squat. i'm in favor of doing this in a measured way. initially there's gonna be nothing in the table so we should see no impact. if we start to see degradation we can always index (and this is something we should do system wide)

Reply:
As per discussion, removing indexes for now